docs/src/ScrollableContainer.doc.js

```diff
@@ -0,0 +1,593 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PageHeader from './components/PageHeader.js';
+import PropTable from './components/PropTable.js';
+import MainSection from './components/MainSection.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="ScrollableContainer"
+    description="ScrollableContainer is used with anchored, hovering components such Flyout, Tooltip, Dropdown, or Typeahead. This is needed for proper positioning when the anchor component could scroll within the viewport."
+  />,
+);
+
+card(
+  <MainSection name="Examples">
+    <MainSection.Subsection
+      description={`The following examples are all built using the \`ScrollableContainer\` component.`}
+    >
+      <MainSection.Card
+        title="Flyout within ScrollableContainer"
```

Comment:
After moving this to become variants, I would recommend putting the title-"Flyout within ScrollableContainer" on the Subsections, not the Card

Reply:
Same would apply for the rest as well - put titles on subsections, not the full-width card

Reply:
DONE